vevestaX/vevesta.py

```diff
@@ -74,12 +76,14 @@ def featureEngineering(self, value):
             if type(value) == pandas.core.frame.DataFrame:
                 cols = value.columns
                 self.__featureEngineering = cols
+                self.__correlation = value.corr(method='pearson')
 
         else:
             if type(value) == pandas.core.frame.DataFrame:
                 cols = value.columns
                 cols = cols.drop(self.dataSourcing)
                 self.__featureEngineering = cols
+                self.__correlation = value.corr(method='pearson')
```

Comment:
this line should be written after if else ..not 2 times once in if and other in else